exchanges/okx/okx.go

```diff
@@ -3147,8 +3147,8 @@ func (ok *Okx) GetCandlestickData(ctx context.Context, instrumentID string, inte
 	var resp [][7]string
 	if limit <= 100 {
 		params.Set("limit", strconv.FormatInt(limit, 10))
-	} else if limit > 100 {
-		return nil, fmt.Errorf("%w limit can not exceed 100", errLimitExceedsMaximumResultPerRequest)
+	} else if limit > 300 {
+		return nil, fmt.Errorf("%w can not exceed 300", errLimitExceedsMaximumResultPerRequest)
```

Comment:
If someone sets their limit between 101-300 the limit field is not set

Reply:
spotto! 😄